dftimewolf/metawolf/metawolf.py

```diff
@@ -257,8 +257,12 @@ def do_set(self, args: argparse.Namespace, _: Any) -> None:
           # Session and recipe settables are not in scope.
           continue
         try:
-          value = input('Set value for parameter {0:s} ({1:s}: {2:s}. Press '
-                        '"enter" to skip or ctrl+d to abort.): '.format(
+          value = input('Set value for{0:s}parameter {1:s} ({2:s}: {3:s}. '
```

Comment:
Are there spaces missing around the `{0:s}`?

Reply:
No, this is on purpose